isis/src/base/objs/Spice/Spice.cpp

```diff
@@ -1017,7 +1054,7 @@ namespace Isis {
   QVariant Spice::readValue(QString key, SpiceValueType type, int index) {
     QVariant result;
 
-    if (m_usingNaif) {
+    if (m_usingNaif and !m_usingAle) {
```

Comment:
Maybe change to `&&` to match the rest of the code? I didn't even realize `and` was valid in C++, fascinating! (Obviously small and it's okay if this doesn't get changed.)